_includes/fancyboxtemplate.html

```diff
@@ -1,4 +1,5 @@
-{{ include.gallery }}
+Gallery: {{ include.gallery }}
+Thumbnail URL: {{ site.galleries[include.gallery].images[0].thumbnail }}
 {% for image in site.galleries[include.gallery].images %}
```

Comment:
Yeah, Liquid doesn't like this either, stumbled on this while developing [`phpDoc.html`](https://github.com/picocms/Pico/blob/7c7030ab6ad8436fc2164c7e71ec718c51297ecb/phpDoc.html#L11-L12). You need a temporary variable (yep, unbelievable stupid, isn't it?):
```twig
{% assign images = site.galleries[include.gallery].images %}
{% for image in images %}
    ...
{% endfor %}
```